x/programs/examples/simulator/testdata/token.yaml

```diff
@@ -0,0 +1,17 @@
+# example of a token program
+name: token program
+description: Deploy and execute token program
+caller_key: alice 
+config:
+  max_memory_pages: 18
```

Comment:
Is there an easy option to use the "chain default"

Reply:
the default now is 16 but I think I am just going to bump it to 18 because everything appears to need that minimum. will dig a bit more before I do that.

Reply:
followup

Reply:
I added support for populating the config via genesis